cmd/fyne/internal/mobile/build_androidapp.go

```diff
@@ -76,11 +76,17 @@ func goAndroidBuild(pkg *packages.Package, bundleID string, androidArchs []strin
 		if err := mkdir(filepath.Dir(libAbsPath)); err != nil {
 			return nil, err
 		}
+		arguments := []string{
+			"-buildmode=c-shared",
+			"-o", libAbsPath,
+		}
+		if release && buildLdflags == "" {
+			arguments = append(arguments, "-ldflags", "-w -s")
```

Comment:
Could you also add a comment above this line and link back to a reference on the release build for android? Basically, explaining why `-ldflags -w -s` in this case?

Reply:
is explaining: <br>
"the other OS (for example windows https://github.com/fyne-io/fyne/blob/7229e889d49c81a83b0b7e09400837f67f6ddad5/cmd/fyne/internal/commands/build.go#L32) has it, we do not need (on release version ) elf and debuh informations, ang we can reduce binary size of apk to 9.5 MB with one arch and aprox. 37.8 MB with all android version"
enough?